nix/pkgs.nix

```diff
@@ -0,0 +1,28 @@
+# This file controls the pinned version of nixpkgs we use for our Nix environment
+# as well as which versions of package we use, including their overrides.
+{ config ? { } }:
+
+let 
+  # For testing local version of nixpkgs
+  #nixpkgsSrc = (import <nixpkgs> { }).lib.cleanSource "/home/jakubgs/work/nixpkgs";
+
+  # We follow the master branch of official nixpkgs.
+  nixpkgsSrc = builtins.fetchTarball {
+    url = "https://github.com/NixOS/nixpkgs/archive/e7603eba51f2c7820c0a182c6bbb351181caa8e7.tar.gz";
```

Comment:
Different form status-mobile one?

Reply:
Good point, we might want to sync together with Sid's upgrade:

* https://github.com/status-im/status-mobile/pull/19369